topiary/rna/gtf.py

```diff
@@ -55,9 +53,11 @@ def load_transcript_fpkm_dict_from_gtf(
         transcript_id: float(fpkm)
         for (transcript_id, fpkm, feature)
         in zip(transcript_ids, fpkm_values, features)
-        if transcript_id is not None
-        and len(transcript_id) > 0
-        and feature == "transcript"
+        if (
```

Comment:
Is this just style?

Reply:
Yeah, I think something about the syntax was bugging Sublime. I can change it back if it's less clear though.

Reply:
No worries either way, just wanted to make sure it wasn't a bug fix